issues.go

```diff
@@ -162,17 +163,32 @@ func issueSearch(ctx context.Context, label, token string, ch chan<- Issue) erro
 			CreatedAt time.Time `json:"created_at"`
 			URL       string    `json:"url"`
 			RepoURL   string    `json:"repository_url"`
+			Labels    []struct {
+				Name  string `json:"name"`
+				Color string `json:"color"`
+			} `json:"labels"`
 		} `json:"items"`
 	}
 	if err := json.NewDecoder(resp.Body).Decode(&data); err != nil {
 		return errors.Wrap(err, "could not decode json")
 	}
 
 	for _, item := range data.Items {
+		issueLabels := make(map[string]string)
+		for _, label := range item.Labels {
+			// Show only labels that are not related to hacktoberfest.
```

Comment:
This isn't working because `"hacktoberfest" != "help wanted"` so they both get through.

We could implement a helper func for this or if we change `labels` from a `[]string` to a `map[string]bool` like we have for organizations and repos then the check would be really easy. You would just have to also tweak the bit where we loop through labels to kick off the workers to be sure it's operating on keys not values.

Reply:
`map[string]bool` seems to be the cleaner solution, so I'll do that and make sure the workers work correctly.